src/app/util/chip-message-send.cpp

```diff
@@ -25,7 +25,8 @@
 #include <assert.h>
 #include <inet/InetLayer.h>
 #include <support/logging/CHIPLogging.h>
-#include <transport/SecureSessionMgr.h> // For SecureSessionMgr
+#include <transport/SecureSessionMgr.h>  // For SecureSessionMgr
```

Comment:
I would remove the comment on this line if we touch it.
I obviously include `SecureSessionMgr.h` for `SecureSessionMgr`.

maybe kMaxTagLen provides a value ... although not as sure even then. I am not used to thse '//For ...' comments after includes.

Reply:
Indeed I just did that because of the existing comment. I'll remove it since I have no doubt that this PR will have other review changes.